content/ux-strat-bootcamp/_index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: UX Strategy Bootcamp
```

Comment:
I would suggest making this a project where people can submit a link that shows that they sucessfully made an account.

```
content_type: project
submission_type: link 
```

Or if you dont need them to submit anything:

```
content_type: topic 
```